magento/magento2
# Fix #14958 - remove sales sequence data on store view delete

In this PR I tried to fix #18466 after it was probably abaddoned.



### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#14958: sale_sequence_* records are not removed on store view deletion

### Manual testing scenarios (*)

1. Delete existing store view
2. Check **sale_sequence_meta** and **sales_sequence_profile** tables - those shouldn't contain any data related to deleted store view.
3. All sequence tables for deleted store view should not exist.

### Contribution checklist (*)
